### DISCUSSION TITLE
Qiqqa is an amazing research tool

### DISCUSSION THREAD

**Discussion by Waxwood** [jimmejardine/qiqqa-open-source#344] [General]
I have been experimenting with Qiqqa for only 2 days and so far, it appears to be an amazing tool. I have tried to keep up with innovative software tools since the early 2000s yet I have no idea why I did not try Qiqqa when I was writing my PhD a few years ago.
Qiqqa's' annotation features allowing the user to highlight text, select sections of text or photos on a PDF and add a keyword to it are amazingly useful and allow it to be used as a simple QDA software tool. I have not seen any other low-cost tool that can do this (I assume how ever that Citavi can do this). 

My only concern is that Qiqqa is not cross-platform and will not run in Linux.
I abandoned Windows years ago and only use it when I must. 
For now, I am running Qiqqa in a virtual Windows 10, but have encountered ongoing crashes that prevent me from seriously using the software. 
Please tell me that a cross-platform version will be released in the near future!!

Thank you for open sourcing Qiqqa- I hope you will be able to build a strong development team. Its an amazing application that more students and researchers should be using.

A message to researchers and graduate students: You really must check out Qiqqa. It's probably the most powerful research tool available (unless you use a pro-level QDA application costing between $400 and $1500 UDS), and it is open source which means that a company can never deny you access to your research project.

**Comment by mauwig** [comment]
I also love Qiqqa and I hope it continues improving. My only problem is the constant crashes, but even with that It's still a powerful tool

**Comment by GerHobbelt** [comment]
Re constant crashes: **if you're using a v83 release**, you might be best served by installing the latest v82 release as that one is known to work well for most people -- the v83 releases are test releases all with some known bugs that can cause frequent crashes.

**Reply by Waxwood** [reply]
Have the developers considered running a crowdfunding campaign to help fund the development of a new cross-platform Qiqqa? If you made a global competition among grad students in social sciences and humanities to raise funds, might that bring funding? Qippa has the potential to become one of the very best research tools-  with some tweaking, it can become the best open-source QDA tool quite capable of competing with software costing many hundreds of dollars. That alone should develop a base of fans willing to donate. I hope :)
